integrations/segment/gitbook-manifest.yaml

```diff
@@ -31,6 +31,8 @@ summary: |
 
     In Segment, you can set up a source to receive the GitBook events. After adding an HTTP API source in Segment, you will see a Write Key. This key is used to identify the data source in GitBook.
 
+    If you are located in the EU, you may need to configure an optional parameter to track events successfully.
```

Comment:
I would maybe bold `**located in the EU**` and maybe add a link to https://segment.com/docs/guides/regional-segment/ for more info.

Also should we maybe mention that the write Key needs to be a EU region writeKey when they use this option since:
> Data sent to the EU endpoint using a US-region write key will get rejected.

https://segment.com/docs/guides/regional-segment/#custom-http-requests